tests/integration-tests/jobs/createJobKubernetes-test.js

```diff
@@ -218,12 +224,150 @@ describe('Create job specific kubernetes tests', async function () {
                         should(relevantJobs[0].id).eql(cronJobId);
                     });
 
+                    it('stop job with experiments', async () => {
+                        nock(kubernetesConfig.kubernetesUrl).delete(`/apis/batch/v1/namespaces/${kubernetesConfig.kubernetesNamespace}/jobs/predator.${firstJobReportId}?propagationPolicy=Foreground`)
+                            .reply(200);
+
+                        nock(kubernetesConfig.kubernetesUrl).delete(`/apis/chaos-mesh.org/v1alpha1/namespaces/${kubernetesConfig.kubernetesNamespace}/podchaos?labelSelector=predator/job-id=${oneTimeJobId}`)
+                            .reply(200);
+
+                        const stopRunResponse = await schedulerRequestCreator.stopRun(oneTimeJobId, firstJobReportId, {
+                            'Content-Type': 'application/json'
+                        });
+
+                        should(stopRunResponse.status).eql(204);
+                        should(nock.pendingMocks().length).eql(0);
+                    });
                     it('Delete jobs', async () => {
                         await schedulerRequestCreator.deleteJobFromScheduler(cronJobId);
                         await schedulerRequestCreator.deleteJobFromScheduler(oneTimeJobId);
                     });
                 });
 
+                describe('Create a job with experiments and use delete all containers to clear them after finished', () => {
+                    let createJobResponse;
+                    let chaosExperimentId;
+
+                    before(async () => {
+                        const chaosExperiment = chaosExperimentsRequestCreator.generateRawChaosExperiment(uuid.v4(), undefined, kubernetesConfig.kubernetesNamespace);
+                        const chaosExperimentResponse = await chaosExperimentsRequestCreator.createChaosExperiment(chaosExperiment, { 'Content-Type': 'application/json' });
+                        chaosExperimentId = chaosExperimentResponse.body.id;
+                    });
+
+                    it('Create job which is one time load_test', async () => {
+                        nock(kubernetesConfig.kubernetesUrl).post(`/apis/batch/v1/namespaces/${kubernetesConfig.kubernetesNamespace}/jobs`)
+                            .reply(200, {
+                                metadata: { name: 'jobName', uid: 'uid' },
+                                namespace: kubernetesConfig.kubernetesNamespace
+                            });
+
+                        const jobBody = {
```

Comment:
let's put job body as const outside the test scope - just to similify the test scope length - can be a function with params

Reply:
is it possible here to take it out of the scope?